src/lib.rs

```diff
@@ -961,6 +961,46 @@ fn lexicographical_topological_sort(
     Ok(PyList::new(py, out_list).into())
 }
 
+#[pyfunction]
+fn graph_greedy_color(
+    py: Python,
+    graph: &graph::PyGraph,
+) -> PyResult<PyObject> {
+    let mut colors: HashMap<usize, usize> = HashMap::new();
+    let mut node_vec: Vec<NodeIndex> = graph.graph.node_indices().collect();
+    let mut sort_map: HashMap<NodeIndex, usize> = HashMap::new();
+    for k in node_vec.iter() {
+        // The intent behind this sort key is to sort nodes in descending order
+        // of degree
```

Comment:
It would be good to remove the comment because the code is revised.

Reply:
Oops I thought that I deleted that when I changed the loop. Yeah I agree it's just confusing now. Fixed in: https://github.com/Qiskit/retworkx/pull/77/commits/2049ddf2746ab3f68aaa14721f730acf613d432b